### ISSUE TITLE
Mask not applied to initial values

### ISSUE THREAD

**Issue by spencerwi** [candreoliveira/ngMask#15]
If a value is already present in a masked field, the mask is not applied until after the user selects the field, then deselects the field.

For example:

![initial-value-not-formatted](https://cloud.githubusercontent.com/assets/2829379/5975871/cd83a27e-a854-11e4-81fd-104a4b394713.gif)

**Comment by candreoliveira** [comment]
Can you use ng-value?

**Comment by spencerwi** [comment]
Wouldn't that defeat the purpose of using ng-model?

**Comment by candreoliveira** [comment]
No. It's the default value of input.

**Comment by spencerwi** [comment]
Ah. I didn't know from the documentation that I needed to add both ng-input and ng-model for the mask to work correctly.

**Comment by spencerwi** [comment]
Tried that, still didn't work. If I have:

```html
<input type="text" mask="(999) 999-9999" 
    ng-value="user.phoneNumber" ng-model="user.phoneNumber">
```

I get the same unformatted field as shown in the GIF above.

**Comment by candreoliveira** [comment]
It was fixed on 3.0.12

**Comment by silverm6** [comment]
This issue still exists in 3.1.1 ..
